src/javascript/app_2/Stores/Modules/Trading/trade_store.js

```diff
@@ -71,7 +71,7 @@ export default class TradeStore extends BaseStore {
     // Start Time
     @observable start_date       = Number(0); // Number(0) refers to 'now'
     @observable start_dates_list = [];
-    @observable start_time       = '12:30';
```

Comment:
how about expiry_time still `9:40` , should it be changed to null as well ?

Reply:
@cakasuma let's work on `expiry_time` in another PR, i think it'll require more work than just changing it to `null`  🙂